setup.py

```diff
@@ -103,6 +103,7 @@ def run(self):
         "tqdm>=4.53.0",
         "pandas>=1.1.5",
         "termcolor>=2.0.0",
+        "matplotlib>=3.5.1",
```

Comment:
make this optional dependency. Use try/imports in your visualize method and wherever else matplotlib is required.

Reply:
Add this optional dependency to `EXTRAS_REQUIRE["all"]` here:

https://github.com/cleanlab/cleanlab/blob/aafcd5469a2c96ebfadcd7ed90b1750220fc3828/setup.py#L39

Then also please make sure `pip install cleanlab[all]` will install both datalab dependencies and matplotlib too.